src/components/TaskInput.js

```diff
@@ -17,9 +17,9 @@ class TaskInput extends React.Component {
     //Function fires when Submit button clicked
     onSubmitClicked() {
         const taskToBeAdded = {
-            id: shortid.generate,
+            id: shortid.generate(),
             description: this.state.taskDescription,
-            completed: true
+            completed: false
```

Comment:
5. Just ensured that new tasks are not marked `false` for their completed status